### ISSUE TITLE
[en] indenting standards

### ISSUE THREAD

**Issue by TiagoSantos81** [languagetool-org/languagetool#1979]
What are the intended indenting criteria to be used in English grammar? This is to avoid further situations as in https://github.com/languagetool-org/languagetool/commit/0c77618b76cfc93c23c403a1aac5822b75592e56#commitcomment-35215566

**Comment by danielnaber** [comment]
4 spaces seems to be the most common. Could you copy your script from Portuguese so it can be used for any language? (I mean with a parameter like `en` so it would work for English then.)

**Comment by TiagoSantos81** [comment]
Sure. I am a bit busy in the coming days, but I will adapt it and copy over once I have a break.

**Comment by TiagoSantos81** [comment]
Closed by de8b195

**Comment by danielnaber** [comment]
I get cases like these:

```
        </rulegroup>
            <rule id="ARE_STILL_THE_SOME" name="are still the some (same)">
```

i.e. `<rule>` is indented even though it's on the same level as `<rulegroup>`. But I guess this cannot be solved with `sed`?
